pyhomematic/_hm.py

```diff
@@ -683,7 +683,7 @@ def proxyInit(self):
                 # at least for home ip, init ccu is not working. Read list devices before "init proxy"
                 # this fix is only applied for HmIP-RF (port 2010), as other interfaces not always implement the listDevice function
                 # and do not show this issue
-                if proxy._remoteport == 2010:
+                if proxy._remoteport == [2010, 32010, 42010]:
```

Comment:
That should be `in`, not `==`.  So like `if element in array`.

Reply:
you are absoletly right - thanks for the hint.